### PR TITLE
Improve runserver docs and hook.

### PR DESCRIPTION
**Pull request by richardbporter**
Can someone give this a try? I can help.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,15 +39,27 @@ BLT provides an automation layer for testing, building, and launching Drupal 8 a
     ```
     $ composer install
     ```
```
Comment (joewhitsitt) [inline README.md:46]: Need to run `blt  blt:init:settings` to generate a local.setttings.php file for each site.

File: README.md [hunk 2]
```diff
@@ -51,15 +39,27 @@ BLT provides an automation layer for testing, building, and launching Drupal 8 a
...
-7. Configure `local.settings.php` to override Lando defaults.
-8. Start the built-in PHP server.
+6. Configure `local.settings.php` to connect to the MySQL instance:
+    ```
+    'database' => $db_name,
+    'username' => 'root',
+    'password' => '',
+    'host' => 'localhost',
+    'port' => '3306',
+    ```
```
Comment (joewhitsitt) [inline README.md]: it wasn't clear if I should leave namespace, driver, prefix defaults.

File: README.md [hunk 3]
```diff
@@ -51,15 +39,27 @@ BLT provides an automation layer for testing, building, and launching Drupal 8 a
...
+6. If the database does not exist, use `drush sql:create` to create it.
+    ```
+    drush -l mysite sql:create
+    ```
```
Comment (joewhitsitt) [inline README.md:50]: I assume this is missing: `blt drupal:sync:all-sites` after the databases have been created

**Review by joewhitsitt — CHANGES_REQUESTED** [review]
If the Lando docs still work, I suggest leaving them in there and updating them one more time with the hiccups that we covered in last week's working session. we can flag lando as unsupported, but i may still use it as it works for me.

File: README.md [hunk 4]
```diff
@@ -22,18 +22,6 @@ This project is based on BLT, an open-source project template and tool that enab
...
-2. Ensure every multisite database exists either through an SQL client or
-   executing `lando drush sql:create` for each multisite.
-3. Run `lando blt drupal:sync:all-sites` to sync all multisites.
-      1. Note: this will discard active configuration!
-4. Profit.
-
-## Runserver
 1. Install Homebrew.
 2. Install PHP 7.2 via Homebrew.
    ```
```
Comment (joewhitsitt) [inline README.md:29]: `brew link php@7.2` and follow steps about putting it first in your PATH

File: README.md [hunk 5]
```diff
@@ -51,15 +42,28 @@ BLT provides an automation layer for testing, building, and launching Drupal 8 a
...
+6. Create all `sites/*/settings/local.settings.php` files.
+    ```
+    blt blt:init:settings
+    ```
+7. Configure each `sites/*/settings/local.settings.php` file to connect to the appropriate MySQL database. You'll need to use the user/password you set during step 3. The host should `localhost`.
+8. If the databases do not exist, use `drush sql:create` to create them. For example:
+    ```
+    drush -l mysite sql:create
+    ```
+9. Start the built-in PHP server.
```
Comment (joewhitsitt) [inline README.md]: Assumes drush is installed globally and Acquia Dev Desktop is not doing shady things in your bash_profile/bashrc

**Comment by richardbporter** [comment]
Raw README: https://github.com/uiowa/uiowa/blob/rs/README.md

**Review by joewhitsitt — APPROVED** [review]

**Comment by joewhitsitt** [comment]
just tested with the latest commits. works as expected. made sure to lando destroy, lando stop, lando power off, quit docker, delete existing databases, and performed a git clean -ffdx before starting.

i already had php@7.2 and drush launcher setup with acquia dev desktop PATH stuff commented out.

**Pull request merged**